src/cache_cf.cc

```diff
@@ -4886,14 +4886,14 @@ parse_UrlHelperTimeout(SquidConfig::UrlHelperTimeout *config)
             else if (strcasecmp(value, "use_configured_response") == 0) {
                 config->action = toutActUseConfiguredResponse;
             } else {
-                debugs(3, DBG_CRITICAL, "FATAL: unsuported \"on_timeout\"  action:" << value);
+                debugs(3, DBG_CRITICAL, "FATAL: unsupported action: on_timeout=" << value);
```

Comment:
The old version was clearer IMO. The new version will be misinterpreted by some that `on_timeout` itself is unsupported.